pkg/controllers/user/helm/common/common.go

```diff
@@ -143,6 +139,52 @@ func InstallCharts(rootDir, port string, obj *v3.App) error {
 	return nil
 }
 
+func GenerateAnswerSetValues(app *v3.App, tempDir string) ([]string, error) {
+	setValues := []string{}
+	// a user-supplied values file will overridden default values.yaml
+	if app.Spec.ValuesYaml != "" {
+		valuesYaml := filepath.Join(tempDir, "answers.yaml")
```

Comment:
can you name this file values.yaml instead? its confusing to call it answers.yaml when it is from the valuesYaml field

Reply:
`values.yaml` is already a default yaml file in each chart, so we can't use it. and the `answers.yaml` is what we have in the past.

Reply:
thats what makes it confusing. its not exactly the same thing as answers.yaml, but if you use the same name, it makes it seem like it is.
How about custom-values.yaml?

Reply:
sure, that's a good choice, will update to it, thanks.